docs/querying/sql-window-functions.md

```diff
@@ -86,9 +91,13 @@ When working with window functions, consider the following:
 
 The OVER clause defines the query windows for window functions as follows:
 - PARTITION BY indicates the dimension that defines the rows within the window
-- ORDER BY specifies the order of the rows within the windows. Currently only ASC, ascending, order works.
-Sometimes windows are called partitions, but don't confuse them with the segment partitioning feature of Druid.
+- ORDER BY specifies the order of the rows within the windows. Currently only ascending order, ASC, works.
+
+:::note
 
+Sometimes windows are called partitions. However, the partitioning for window functions are a shuffle/partition of the result set that happens at query time and is not to be confused with Druid's segment partitioning feature which partitions data at ingest time.
```

Comment:
Recommend removing slashes for clarity: https://developers.google.com/style/slashes#slashes-with-alternatives